data/workshops.ts

```diff
@@ -5,6 +5,7 @@ const WORKSHOPS = [
   {
     slug: "aha-stack",
     published: true,
+    registrationOpen: false,
```

Comment:
@necccc it would make sense to get this data from the api too

Reply:
Done!

``` 
{"RISINGSTACK":{"seats":10,"taken":0,"open":true},"SUPERCHARGE":{"seats":10,"taken":0,"open":true}}
```